### PR TITLE
Spell check: Update + optimization

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: libse/SpellCheck/SpellCheckWordLists.cs [hunk 1]
```diff
@@ -184,7 +184,6 @@ private void SaveUseAlwaysList(string newKey = null, string newValue = null, str
             {
                 _useAlwaysList.Remove(oldKey.Trim());
             }
-            _skipAllList = new HashSet<string>(_skipAllList.OrderBy(p => p).ToList());
```
Comment (ivandrofly) [inline libse/SpellCheck/SpellCheckWordLists.cs]: no need for this...

**Comment by ivandrofly** [comment]
review? /cc @xylographe

**Comment by niksedk** [comment]
I have some problems with my house due to broken water pipes :(
My dev computer is packed away and I don't really have much time for anything else for the next 2-4 months. so large PRs will have to wait, sorry.

**Comment by ivandrofly** [comment]
> I have some problems with my house due to broken water pipes :(
> My dev computer is packed away and I don't really have much time for anything else for the next 2-4 months. so large PRs will have to wait, sorry.

Oh sure... I'm sorry btw!

**Comment by darnn** [comment]
Hang in there!

**Pull request closed without merging**